dashboard/app/models/cap/user_event.rb

```diff
@@ -29,6 +29,7 @@ class UserEvent < ApplicationRecord
       ACCOUNT_LOCKING = 'account_locking'.freeze,
       ACCOUNT_PURGING = 'account_purging'.freeze,
       COMPLIANCE_REMOVING = 'compliance_removing'.freeze,
+      SECTIONS_WARNING_EMAIL_SENT = 'sections_warning_email_sent'.freeze
```

Comment:
I’m not sure it’s the best idea to report this event here, since teachers aren’t covered by any Child Account Policies.
https://github.com/code-dot-org/code-dot-org/blob/bb3f86254ad0fe299a12894ecd5ba58f71f9178f/dashboard/app/models/cap/user_event.rb#L8

Reply:
Chatted with Artem offline about this - I think the hesitation to add this event to the CAP table is correct. This is technically an action that goes out to teachers, so we'd need to stick some kind of json in the event to link the teacher email event back to the section the student was enrolled in. Taking another step back - the main motivation of capturing this event is to see if sending teacher emails has a meaningful impact on student permission conversions. Accordingly we should just be reporting this to our 3rd party analytics provider (currently statsig) - and possibly even set it up as an AB test.